doc/tutorials/open_vswitch.rst

```diff
@@ -0,0 +1,155 @@
+.. _open_vswitch:
+
+Using OVS bridge instead of ACRN bridge
+#######################################
+Hypervisors need the ability to bridge network traffic between VMs
+and with the outside world. This tutorial describes how to
+use Open Virtual Switch (OVS) bridge instead of ACRN
+bridge for this purpose.
+
+.. note::
+   OVS is integrated in the ClearLinux ``service-os``
```

Comment:
Rather than "integrated", I would say it is provided as part of the `service-os` bundle (but it's not active/configured by default).